packages/aws-cdk-lib/aws-ses/lib/configuration-set.ts

```diff
@@ -80,6 +80,15 @@ export interface ConfigurationSetProps {
    * @default - VDM options not configured at the configuration set level. In this case, use account level settings. (To set the account level settings using CDK, use the `VdmAttributes` Construct.)
    */
   readonly vdmOptions?: VdmOptions;
+
+  /**
+   * The maximum amount of time that Amazon SES API v2 will attempt delivery of email.
+   *
+   * This value must be greater than or equal to 5 minutes and less than or equal to 14 hours.
+   *
+   * @default - 14 hours
```

Comment:
nit: I'd be specific here, `@default undefined - SES defaults to 14 hours`

Reply:
I've updated!